library/Icingadb/Model/History.php

```diff
@@ -63,7 +63,7 @@ public function getColumnDefinitions()
 
     public function getDefaultSort()
     {
-        return 'history.event_time desc';
+        return 'history.event_time desc, history.event_type desc';
```

Comment:
Use this sort also for `$sortControl` in `HistoryController:indexAction` or else as @julianbrost mentioned sort dropdown shows "Custom"